scalableminds/webknossos
# Make access token check explicit in data/tracing stores

The implicit accessing of GET parameters via getQueryString led to confusion when reading the code and to them missing from the swagger docs. This PR changes them to explicit GET parameters (usually tokens identifying the user, or keys identifying datastores/tracingstores).

### URL of deployed dev instance (used for testing):
- https://explicittokencheck.webknossos.xyz

### Steps to test:
- Tracing saving, upload/download, data viewing should still work both if logged in and (viewing public datasets) if logged out

### Issues:
- fixes #5875
- fixes #5864 

Follow-up for Filters queries (the one remaining instance of `getQueryString` after this): https://github.com/scalableminds/webknossos/issues/5891

------
